### ISSUE TITLE
The add-on (version 2.8) cannot be installed from the Add-on Store

### ISSUE THREAD

**Issue by CyrilleB79** [mahmoodhozhabri/VisionAssistantPro#29]
### Issue
I have version 2.7 installed (externally). NVDA Add-on Store offers me to update to version 2.8. However, when I try to update, the update fails because the checksum is not the same.

The most likely reason is that you have updated the 2.8 release after you have already submitted it to the Add-on Store.

### Solution

Create a version 2.9, or 2.8.1, instead.

And in the future, do never modify a release after you have submitted to the Add-on Store. Instead, create a new release if you have something to fix.
